TDD_Training/TDD_Training/Program.cs

```diff
@@ -10,12 +10,19 @@ public static void Main()
             try
             {
                 var inputNumber = int.Parse(Console.ReadLine());
-                FizzBuzzService fizzBuzzService = new FizzBuzzService();
-                fizzBuzzService.CheckFizzBuzz(inputNumber);
+                if(inputNumber <= 0)
```

Comment:
Nope. Not here. All the logic should reside inside your Service. Please move it there. and write necessary test cases as well.
you Main() method should just be calling them.